modules/travel_pay/app/services/travel_pay/service.rb

```diff
@@ -0,0 +1,4 @@
+module TravelPay
```

Comment:
I think this might be vestigial? (Left over from when we broke the generic `service` out into more focused services?